zone2/main.c

```diff
@@ -1,27 +1,126 @@
-/* Copyright(C) 2018 Hex Five Security, Inc. - All Rights Reserved */
+/*
+ * Copyright(C) 2018 Hex Five Security, Inc. - Some rights reserved.
+ *
+ * This file is part of multizone-sdk/zone2.
+ *
+ * multizone-sdk/zone2 is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * multizone-sdk/zone2 is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with multizone-sdk/zone2. If not, see <https://www.gnu.org/licenses/>.
+ */
 
 #include <platform.h>
 #include <libhexfive.h>
 
-int main (void){
+#define RV32
```

Comment:
Needed because pico_config.h includes all the possible configurations, without flattening them, so we would end up with pico_posix.h which declares `pico_zalloc`, `pico_free`, `PICO_TIME_MS`, `PICO_TIME` and `PICO_IDLE` differently (i.e. `static inline ...`).